tokens/src/core/alias/size.json

```diff
@@ -1,11 +1,11 @@
 {
   "size": {
     "border": {
-      "width": { "value": "1px", "type": "dimension", "source": "$border-width" },
+      "width": { "value": "1px", "type": "dimension", "source": "$border-width", "description": "Border width value." },
```

Comment:
Is the plan to have *every* token include a description? I don't think "Border width value" provides any any additional context over `border-width` or `side.border.width`.

If the idea is to be able to rely exclusively on descriptions then I understand having this, but I would suggest simply "Border width" without "value"

Reply:
I *think* eventually we want to have description for every token, but not in this PR. At some point in the future we will have a UI to update token values without explicitly opening a PR (e.g., designers could update token values without engineers' help), and ideally we'd also want to have some kind of a description for tokens in that UI, our idea is to store and take such descriptions from the JSON files.

I agree though that most of the token names are self-explanatory and maybe we don't need descriptions for every token, on the other hand we could add it just to be consistent.

Also, yeah, `value` is redundant in the description, I'd suggest `Default border with` instead

Reply:
Changed to `Default border width`